bedrock/firefox/templates/firefox/privacy_tour/index.html

```diff
@@ -11,6 +11,20 @@
 
 {% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
 
+{% block page_title %}
```

Comment:
I would recommend removing this from the PR, as it's just creating superfluous changes.